pkg/config/config.go

```diff
@@ -677,6 +680,23 @@ func (bp *Blueprint) checkBlueprintName() error {
 	return nil
 }
 
+// checkToolkitModulesUrlAndVersion returns an error if either
+// toolkit_modules_url or toolkit_modules_version is
+// exclsuively supplied (i.e., one is present, but the other is missing).
+func (bp *Blueprint) checkToolkitModulesUrlAndVersion() error {
+	if bp.ToolkitModulesURL == "" && bp.ToolkitModulesVersion != "" {
+		return BpError{Root.ToolkitModulesURL, HintError{
```

Comment:
It needs to be `Root.ToolkitModulesVersion`, since `bp.ToolkitModulesURL` is not defined there is no place in blueprint file to point to.